doc/README.md

```diff
@@ -9,4 +9,5 @@ Bundle with utility classes.
 * [Fixture Builder](fixture-builder.md) - Quick way to type fixtures
 * [Menu Tree](menu-tree.md) - Menu tree and repair command  
 * [Translations](translations.md) - Translate from javascript and `zz` locale
-* [LIPP Imagine](lipp-imagine.md) - Match filter for liip imagine 
+* [LIPP Imagine](lipp-imagine.md) - Match filter for liip imagine
```

Comment:
LIPP of LIIP? :trollface:

Reply:
_Weet het niet aan jou ligt_

Reply:
Kom wel meer dingen tegen die een beetje iffy zijn... maar zit is een oude versie van de bundle, die doe er expres zo min mogelijk in om merge conflicten te voorkomen.  Wel een goede om in versie 8 ff goed te zetten